src/ai/backend/web/proxy.py

```diff
@@ -399,6 +400,25 @@ async def websocket_handler(request, *, is_anonymous=False) -> web.StreamRespons
                 content_type=request.content_type,
                 override_api_version=request_api_version,
             )
+            if proxy_path == "pipeline":
+                aiohttp_session = request.cookies.get("AIOHTTP_SESSION")
+                if not (sso_token := request.headers.get("X-BackendAI-SSO")):
+                    jwt_secret = request.app["config"]["pipeline"]["jwt"]["secret"]
+                    now = datetime.now(tz=timezone(timedelta(hours=9)))
```

Comment:
As Backend.AI is not only served based on KST (yes, we are international!) it is a bad idea to hardcode the timezone like this. When adding `.astimezone()` to the end of `now()` call automatically detects time timezone set on host machine and updates the datetime object ([ref](https://docs.python.org/3.11/library/datetime.html#datetime.datetime.astimezone)), let's update this code to adapt these behavior.